doc/user/content/guides/postgres-cloud.md

```diff
@@ -123,6 +121,44 @@ As a user with the `cloudsqlsuperuser` role, make these changes to the upstream
 
 For more information, see the [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance) documentation.
 
+## Azure Database for PostgreSQL
+
+Before you start, note that a database restart will be required after making changes to the database.
+
+1. The Materialize replica will need access to connect to the upstream database. In your Azure portal, go to the Azure Database for PostgreSQL instance and under the "Connections security" section add your [Materialize instance's IP address](/cloud/security/#static-ip-addresses) to the allowed IP addresses list and click on the "Save" button.
+
+1. Enable Logical Replication by going to the Azure Database for PostgreSQL instance, then under the "Replication" section click the "Enable Logical Replication" toggle button and click on "Save".
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
```

Comment:
Is it not possible to `CREATE PUBLICATION mz_source FOR ALL TABLES;` due to user constraints? I see this is being used for e.g. Cloud SQL, where you also don't get superuser rights; maybe worth double checking and correcting for other services.

Reply:
Yep, it is mentioned a couple of lines bellow.
Will try to replicate this with Cloud SQL, it seems to take ages to spin up an instance though
<img width="439" alt="image" src="https://user-images.githubusercontent.com/21223421/159292242-57f7d40e-e3ea-4124-ac24-bc048d71cb18.png">

Reply:
Ok so this does not affect Cloud SQL, as the users created with built-in authentication are granted the `cloudsqlsuperuser` role, and have the same set of attributes as the postgres user which is not the case with the `doadmin` user.

Reply:
Thanks for double-checking!